web/cypress/e2e/pages/study-management.cy.ts

```diff
@@ -153,8 +153,51 @@ describe("Study Assets Management", () => {
 
     cy.visit("/studies/manage?studyId=123456789");
 
-    cy.mockStudyAssetsWithSample();
+    cy.mockInformationAssetsWithSample();
     cy.wait("@getAssetsWithSample");
     cy.contains("Sample Asset Title 1").should("be.visible");
+    cy.contains("requires a contract").should("be.visible");
+  });
+});
+
+describe("Study Updates", () => {
+  beforeEach(() => {
+    cy.mockAuthAsStudyOwner();
+
+    cy.mockStudiesWithNewStudy();
+    cy.mockStudyAccess();
+    cy.mockStudyAgreementText();
+    cy.mockStudyAgreementsConfirmed();
+    cy.mockAssetCreation();
+    cy.mockInformationAssetsWithSample();
+    cy.mockContractsWithSample();
+
+    cy.visit("/studies/manage?studyId=123456789");
+    cy.waitForAuth();
+    cy.wait("@getStudyById");
+    cy.wait("@getStudyAgreementText");
+    cy.wait("@getStudyAgreementsConfirmed");
+    cy.wait("@getAssetsWithSample");
+    cy.wait("@getContractsWithSample");
+  });
+
+  it("should successfully update a study as its owner", () => {
+    cy.mockStudyUpdate();
+    cy.contains("My New Test Study").should("be.visible");
+    cy.contains("Manage Study").click();
+
+    cy.contains("Risk Score").should("be.visible");
+    cy.contains("Additional Information").should("be.visible");
+    cy.contains("Mark Ready for Review").should("be.visible");
+    cy.contains("Edit Study").click();
+
+    // edit the description and submit
+    cy.get("textarea#description").type("My New Test Study Description");
+    cy.get("[data-cy='next']").click();
+    cy.get("[data-cy='next']").click();
+
+    cy.get("button[type='submit']").click();
+
+    cy.wait("@updateStudy");
```

Comment:
I think as soon as this PR is merged we should prioritise https://github.com/orgs/ucl-arc-tre/projects/1/views/15?pane=issue&itemId=136189709&issue=ucl-arc-tre%7Cportal%7C354 to allow more robust testing of the functionality. Then we should come back and replace some of these tests with true e2e versions.